src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SecurityServerTlsCertificate/GenerateTlsAndCertificateDialog.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <simpleDialog
+    :dialog="dialog"
+    title="ssTlsCertificate.generateTlsAndCertificateDialog.title"
+    saveButtonText="action.confirm"
+    :showClose="false"
+    @save="save"
+    @cancel="$emit('cancel')"
+    :loading="loading"
+  >
+    <div slot="content">
+      <p data-test="generate-tls-and-certificate-dialog-explanation-text">
+        {{ $t('ssTlsCertificate.generateTlsAndCertificateDialog.explanation') }}
+      </p>
+      <p data-test="generate-tls-and-certificate-dialog-confirmation-text">
+        {{
+          $t('ssTlsCertificate.generateTlsAndCertificateDialog.confirmation')
+        }}
+      </p>
+    </div>
+  </simpleDialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import SimpleDialog from '@/components/ui/SimpleDialog.vue';
+import * as api from '@/util/api';
+
+export default Vue.extend({
+  components: { SimpleDialog },
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  props: {
+    dialog: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  methods: {
+    save() {
+      this.loading = true;
+      api
+        .post('/system/certificate', {})
+        .then((_) => {
```

Comment:
Whi there is _  in here?

Reply:
Actually not needed in this instance. I am used to writing .then(_ => {}) in which case "_" marks the unused parameter, otherwise it will be an error or a warning. 

Since the linter changes it to (_) => {} then it is no longer needed.

Fixed.